frontend/src/components/Export/Pill.svelte

```diff
@@ -42,7 +42,7 @@
     }
 
     .error {
-        background-color: #c2586c;
-        color: #b50d0d;
+        background-color: #A9282E;
+        color: #FFF0F1;
```

Comment:
pretty sure this change was made in [INTG-2665](https://safetyculture.atlassian.net/browse/INTG-2665)